### PR TITLE
article(angular-bower): fix typos and minor errors

### PR DESCRIPTION
**Pull request by gkalpak**

### CODE REVIEW

File: jade/blog/angular-bower.md [hunk 1]
```diff
@@ -271,19 +271,19 @@ I like giving an `<optional-thing-type>` for directives and filters when it's no
 
 I like giving an `<optional-namespace>` when I have a group of similar components, as when writing wrappers for PhoneGap.
 
-I wouldn't bother creating a separate "build repo" for most projects. If the build step is "[ngmin](https://github.com/btford/ngmin) + uglify," users of this component can have their project's build system take care of it. If you have something more involved, then I think the build/release repo is a fine approach.
+I wouldn't bother creating a separate "build repo" for most projects. If the build step is "[ngmin](https://github.com/btford/ngmin)/[ng-annotate](https://github.com/olov/ng-annotate) + uglify," users of this component can have their project's build system take care of it. If you have something more involved, then I think the build/release repo is a fine approach.
```
Comment (gkalpak) [inline jade/blog/angular-bower.md:274]: Or maybe you want to remove `ngmin` altogether (since it is deprecated).

File: jade/blog/angular-bower.md [hunk 2]
```diff
@@ -328,16 +328,16 @@ angular.module('my-d3-module', []).factory('d3', function () {
...
-4. After `3`, the developer for `d3` releases a new version of `d3` with the same version number as `3`'s effective fork of `d3`, creating confusion for users of `angular-d3` and `d3`.
+4. Later on, the developer for `d3` releases a new version of `d3` with the same version number as the previously updated effective fork of `d3`, creating confusion for users of `angular-d3` and `d3`.
 
 The "right" way to use `d3` right now is to include it as a dependency of some directive that accesses the library directly off of `window.d3`.
 
 **tl;dr: please don't create wrapper modules for components that expose their APIs on `window.whatever`. Just use the API off `window` for now. If Angular needs `$apply` for some `foo-lib` library, feel free to write an adapter, but the adapter should depend on `foo-lib`, not include the files for it.**
 
-The ideal solution to this problem to be standardized ES6/ES7/ESEventually modules and Web Components, but we still don't know when something like that will land.
+The ideal solution to this problem would be standardized ES6/ES7/ESEventually modules and Web Components, but we still don't know when something like that will land.
```
Comment (gkalpak) [inline jade/blog/angular-bower.md:340]: Maybe a note about modules landing on the upcoming ES6 might be in place :)

**Comment by gkalpak** [comment]
And perhaps you might want to mention that the "soon" has come (in respect to [(like `component.json` will soon be `bower.json`)](https://github.com/btford/briantford.com/pull/25/files#diff-355cabe61a2a33e8c84b3ca0d2ad9bb4R11)).
